src/jit/codegen.h

```diff
@@ -465,11 +465,11 @@ protected :
                                     IL_OFFSETX                                    ilOffset);
 
 
+#if defined(_TARGET_ARM_)
     //
```

Comment:
Where is the best place to put this new #define? I've added it to target.h, because I know that gets included by everything, but I'm not sure it's the right place.